src/TODO_factory.js

```diff
@@ -0,0 +1,20 @@
+const todo = (
+	title,
+	description,
+	dueDate,
+	priority = 'NOT IMPORTANT',
+	projectName = 'GENERAL',
+	status = 'INCOMPLETE',
+) => {
+		return ({
```

Comment:
The reason why this error is here it's because of how you are doing your `return` statement. Why do you need to wrap the object in a `()`? 

If you really want to wrap the returned object with a `()` then you can just discard the `return` keyword and just do an "implicit return" as such:

```javascript
const todo = (
  title,
  description,
  dueDate,
  priority = 'NOT IMPORTANT',
  projectName = 'GENERAL',
  status = 'INCOMPLETE',
) => ({
  title,
  description,
  dueDate,
  priority,
  projectName,
  status,
});
```